portfolio/src/main/java/com/google/sps/servlets/GlobalWarmingDataServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns global warming sentiment data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
+@WebServlet("/global-warming-sentiment-data")
+public class GlobalWarmingDataServlet extends HttpServlet {
+
+  private LinkedHashMap<String, Integer> globalWarmingSentiment = new LinkedHashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/global-warming-sentiment-data.csv"));
+    
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
+
+      String tweet = (String) cells[0];
+      Integer existence = 0;
+      if(((String) cells[1]).equals("Yes")){
```

Comment:
Whitespace.

Reply:
What does "Yes" / "No" mean?

Reply:
On my step_project.html page, I have a description displayed to the user about the meaning behind the "Yes" and "No". Maybe I can leave a comment here as well saying what they mean? Essentially, for all the tweets in the study corresponding to the dataset I chose, "Yes" indicates that the tweet confirms the existence of climate change, and "No" indicates that the tweet denies the existence of climate change.

Reply:
I think that its good to add it here as well as the html. That way when people look at ur code (either js or html) they will understand.